android/app/src/main/res/values-es/strings.xml

```diff
@@ -0,0 +1,88 @@
+<string name="control_check">Ctrl</string>
+<string name="delete_connection">Eliminar</string>
+<string name="disconnect">Desconectar</string>
+<string name="toggle_framebuffer_update">Cambiar la vista</string>
+<string name="toggle_mouse_buttons">Cambiar los botones del ratón</string>
+<string name="toggle_pointer_highlight">Cambiar el resaltado del puntero</string>
+<string name="toggle_keyboard">Cambiar el teclado</string>
+
+<string name="export_settings">Exportar</string>
+<string name="export_settings_path">Exportar a un archivo</string>
+<string name="database_error_open">!Error al abrir la base de datos!</string>
+<string name="import_export_settings">Importar/Exportar</string>
+<string name="import_settings">Importar</string>
+<string name="import_settings_file_url">Importar desde un archivo o URL</string>
+    <string name="import_error_url">Se ha dado una URL incorrecta:</string>
+    <string name="import_error_io">Configuración de la lectura de errores de I/O</string>
+    <string name="import_error_xml">Configuración de la lectura de errores deformato o XML </string>
+
+<string name="info">Información de la conexión</string>
+<string name="meta_key_title">Enviar combinación de teclas</string>
+<string name="new_connection">Conexión nueva</string>
+<string name="nickname_caption">Marcadores</string>
+<string name="discovered_servers">Servidores descubiertos</string>
+<string name="bookmarks">Marcadores</string>
+<string name="savebookmark_button">Guardar como marcador</string>
+<string name="bookmark_saved">¡Marcador guardado!</string>
+<string name="bookmark_invalid">¡Esto ya no es un marcador válido!</string>
+<string name="password_caption">Contraseñ</string>
+<string name="keep_password_button">Mantener</string>
+<string name="port_caption">Puerto</string>
+<string name="repeater_caption">ID en el repetidor</string>
+<string name="repeater_hint">Identificación del servidor cuando se conecta a un repetidor UltraVNC</string>
+<string name="save_as_copy">Guardar como copia</string>
+    <string name="credentials_needed_title">Se necesitan credenciales...</string>
+<!-- Send selected metakey -->
+<string name="send_button">Enviar</string>
+<string name="send_key_again">Envíar la combinación de teclas de nuevo</string>
+<string name="shift_check">Cambio</string>
+<string name="shortcut_label">Atajo de la conexión VNC</string>
+    <string name="shortcut_no_entries">No hay marcadores todavía</string>
+<string name="special_keys">Cpmbinación de teclas</string>
+<string name="username_caption">Usuario</string>
+<string name="username_hint">Para la autentificación de Windows/Mac</string>
+<string name="editbookmark_title">Editar Marcadores</string>
+<string name="enterbookmarkname">Por favor, introducir un nombre de marcador...</string>
+<string name="bookmark_save_question">¿Quiere añadir este servidor a los marcadores?</string>
+<string name="bookmark_delete_question">¿Estás seguro de que quiere borrar este marcador?</string>
```

Comment:
Maybe we should stick to 'usted' like in line 60? Right now it's a mix of "tu" and "usted", right?

Reply:
Right, it's a mistake, we'll stick with "usted"